src/Stratis.Bitcoin.Features.Miner/Controllers/MiningApiController.cs

```diff
@@ -89,26 +90,35 @@ public IActionResult Generate([FromBody]MiningRequest request)
             }
             catch (Exception e)
             {
-                this.logger.LogError("Exception occurred: {0}", e.ToString());
+                this.logger.LogError(ExceptionOccurredMessage, e.ToString());
                 return ErrorHelpers.BuildErrorResponse(HttpStatusCode.BadRequest, e.Message, e.ToString());
             }
         }
-   
+
         /// <summary>
         /// Finds first available wallet and its account.
         /// </summary>
         /// <returns>Reference to wallet account.</returns>
         private WalletAccountReference GetAccount()
         {
+            const string noWalletMessage = "No wallet found";
+            const string noAAccountMessage = "No account found on wallet";
```

Comment:
`noAAccountMessage ` should be `noAccountMessage` ;)

Reply:
Oops! Done.